### ISSUE TITLE
"Share Button" opens empty PopUp Menu

### ISSUE THREAD

**Issue by AlbinoGeek** [fyne-io/apps.fyne.io#15]
On an app listing page (again, using this as an example:)
https://apps.fyne.io/apps/com.GitHub.mehrdad-dev.jami.html

There is a share icon in the top right of the main content container.

Clicking this currently opens an empty PopUp.

**Comment by andydotxyz** [comment]
I see the following:

<img width="345" alt="Screenshot 2021-01-29 at 18 41 44" src="https://user-images.githubusercontent.com/294436/106314460-aab0df00-6261-11eb-8c38-2370324f8bde.png">

What do you see?

**Comment by AlbinoGeek** [comment]
![image](https://user-images.githubusercontent.com/1910461/106347376-c7094780-6272-11eb-903e-0f747a6caa02.png)
I see... a lot of console security warnings/errors, and the HTML items in that dropdown are commented out on pageload.

That means (somehow) these links are triggering my security extensions.  The only obvious block I can see on that page is a reference to ` cdn.viglink.com ` which is on my tracker blacklist.  The above is replicated in Firefox 84.0.2 (64-bit) for Fedora 33 Workstation

My Firefox Security Extensions:

- AdBlock Plus Fork (Disabled for `http*://*.fyne.io/*`
- Decentraleyes (0 filters applied on `https://apps.fyne.io/*`)
- Privacy Badger/Possom Fork (1 learned filter (`viglink`))

I also have Firefox' built-in "Enhanced Trackers Blocking" enabled.

---

Chrome with a clean profile produces the same empty share menu, and new console errors:

![image](https://user-images.githubusercontent.com/1910461/106347526-05533680-6274-11eb-9188-c8eadf7c9269.png)

---

(Again, not a fan of pixel tracking, why is this here, I'm opening a new issue for that...)

**Comment by andydotxyz** [comment]
Here Chrome does not do this. It looks like you have google analytics blocked.
We use google analytics to try and get a feel for how popular the pages and apps are.

**Comment by Jacalz** [comment]
Could we switch to something else than Google analytics instead? It is a nightmare for privacy. I’d recommend using this instead: https://brave.com/privacy-preserving-product-analytics-p3a/

**Comment by andydotxyz** [comment]
Any hosted service that is available to us for free would be an acceptable replacement for sure.

**Comment by AlbinoGeek** [comment]
`viglink` isn't `googletagmanager` isnt `analytics.google.com` -- the latter two I have enabled/unblocked.
[ The last two are only the same thing if you are using Google's new non-analytics-specific tracking code. ]

Which, yes, nightmare to privacy. But.. `viglink` is a larger nightmare to privacy, hence, concerned that it's there.

**Comment by andydotxyz** [comment]
It seems to be called by the Disqus plugin.
If there is a better free alternative we could switch it out - there are not too many comments that would be lost at this stage.
Ideally it would have a jekyll plugin so we don't have to code it up!

**Comment by AlbinoGeek** [comment]
Personally, I like Plausible[^1] or GoatAnalytics[^2] when it comes to privacy-focused analytics platforms.

Plausible[^1] works out to roughly $4/mo if you choose not to host it yourself (It's Open Source.)

GoatAnalytics[^2] is free for non-commercial use on metrics for the first 100k pageviews per month.

---

I will reply again when I find a suitable alternative for Disqus.

[^1]: https://plausible.io/
[^2]: https://github.com/zgoat/goatcounter

**Comment by AlbinoGeek** [comment]
Alright, unfortunately there really isn't a suitable replacement if we don't want to do data-basing or otherwise add an external dependency, even a free public hosted database -- it's just more work for the team than I'd like to propose.

Sorry about all that!

I will fork the previous message out to its own issue in regards to Analytics privacy.
